elements/pfe-progress-steps/src/pfe-progress-steps-item.scss

```diff
@@ -201,14 +202,17 @@ $LOCAL-VARIABLES: map-deep-merge($LOCAL-VARIABLES, (
 
     @each $state in (hover, focus, visited) {
       :host([hasLink]:#{$state}) & {
+        // MP color on hover should be #0066CC
         color: pfe-local(Color, $region: title, $fallback: pfe-broadcasted(link--#{$state}));
```

Comment:
If this is a link, the hover color should be the link hover color or focus color for consistency.  #06c is the link color default.